doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -74,16 +74,14 @@ As `req.inf.ntw.14` mandates the architecture must enable the integration of dif
 The architecture must support networking for telecom equipments in an environment where the networks of the CNF-s are set up by the network adminisrators of the telecom operator. This is why, as `req.inf.ntw.10` requires, the architecture must provide a set of abstract management API-s to manage the network connectivity of the CNF pods.
 The API must support multiple tenants and must require elevated acces rights to manipulate infrastructure related API objects as these operations require reconfiguration of the physical network infrastructure.
 
-To fullfill the requirements of `req.inf.acc.02` the architecture must support the usage of device plugins via the Device Plugin API, also the alignment of the devices and the CPU topology must be supported using the [Topology Manager](https://kubernetes.io/docs/tasks/administer-cluster/topology-manager/).
+To fullfill the requirements of `req.inf.acc.02` the architecture must support the usage of device plugins via the Device Plugin API, also the alignment of the devices, CPU topology and hugepages must be supported using the [Topology Manager](https://kubernetes.io/docs/tasks/administer-cluster/topology-manager/).
 
 The architecture must support both IPv4, IPv6 and dual stack interfaces of the workloads.
  > How about IPv6 support of the control plane?
 
-As Kubernetes Ingress and Services have no support for all the protocols needed in telecommunication environments (Diameter, SIP, LDAP, etc) and their capacity is limited the architecture must enable the usage of alternative load balancers, like external or built into the application.
+As Kubernetes Ingress, Egress and Services have no support for all the protocols needed in telecommunication environments (Diameter, SIP, LDAP, etc) and their capacity is limited the architecture must enable the usage of alternative load balancers, like external or built into the application.
 
-Well known service meshes are "application service meshes" and therefore deal with layer 7 - application - protocols, like HTTP, only therefore their support is not required in the architecture.
-
-As the most recent version of Network Service Mesh is v0.1.0 at the time of specifying RA2 in Snezka release it is not required in the architecture.
+Well known service meshes are "application service meshes" and therefore deal with layer 7 - application - protocols (eg.: HTTP), only therefore their support is not required in the architecture.
```

Comment:
Suggest change from:
"Well known service meshes are "application service meshes" and therefore deal with layer 7 - application - protocols (eg.: HTTP), only therefore their support is not required in the architecture."

to 

"The well known service meshes are "application service meshes" and deal with the application layer 7 protocols (eg.: HTTP) only. Therefore, their support is not required in the architecture."